libs/wingsdk/src/utils/aws/aws-names.ts

```diff
@@ -0,0 +1,76 @@
+import { CaseConventions, NameOptions, ResourceType } from "../resource-names";
+
+export const awsResourceNameProps = (
```

Comment:
Sorry for being a bit picky about this, but kind of feels like each one of these can go into a local `const` next to where it is used instead of in a centralized location.

Reply:
Ohhhh you mean which resource pass its options

This makes sense and all the logic regarding the resource will be in the resource itself